termux-exec.c

```diff
@@ -231,8 +236,33 @@ int execve(const char* filename, char* const* argv, char* const* envp)
 		}
 	}
 
-	int ret = real_execve(filename, argv, envp);
+	int ret = real_exec_fn(filename, argv, envp);
 	free(new_argv);
 	free(new_envp);
 	return ret;
+	
+}
+
+int _execvp(const char* filename, char* const* argv, char* const* envp)
+{
+        (void)envp;
+        int (*real_execvp) (const char*, char* const[]) = dlsym(RTLD_NEXT, "execvp");
+        return real_execvp(filename, argv);;
+}
+
+int _execve(const char* filename, char* const* argv, char* const* envp)
+{
+        int (*real_execve)(const char*, char* const[], char* const[]) = dlsym(RTLD_NEXT, "execve");
+        return real_execve(filename, argv, envp);
```

Comment:
We don't do any special things here. Then why bother intercepting this weak symbol in the first place?

Reply:
This function is unnecessary. Removed.